middleware/cache/README.md

```diff
@@ -131,6 +131,14 @@ type Config struct {
 	//
 	// Default: []string{fiber.MethodGet, fiber.MethodHead}
 	Methods []string
+
+	// Allows you to use no-cache and no-store request directives
+	// https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Cache-Control
+	// 
+	// Default: func(c *fiber.Ctx, directive string) bool {
+	// 	return strings.Contains(c.Get(fiber.HeaderCacheControl), directive)
+	// },
+	RequestDirective func(c *fiber.Ctx, directive string) bool
```

Comment:
@marcmartin13 
not sure yet if we really should expose the RequestDirective method

what can i configure as a consumer of the middleware other than what is configured there, if this method is used in two different places

Reply:
oh right. i'll make it private

Reply:
@ReneWerner87 
oh right, I'll make it private for now